src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -2648,11 +2623,11 @@ namespace DataHeatBalance {
     extern Array1D<ScreenTransData> ScreenTrans;
     extern Array1D<ZoneCatEUseData> ZoneIntEEuse;
     extern Array1D<RefrigCaseCreditData> RefrigCaseCredit;
-    extern Array1D<HeatReclaimRefrigeratedRackData> HeatReclaimRefrigeratedRack;
+    extern Array1D<HeatReclaimDataBase> HeatReclaimRefrigeratedRack;
     extern Array1D<HeatReclaimRefrigCondenserData> HeatReclaimRefrigCondenser;
-    extern Array1D<HeatReclaimDXCoilData> HeatReclaimDXCoil;
-    extern Array1D<HeatReclaimDXCoilData> HeatReclaimVS_DXCoil;
-    extern Array1D<HeatReclaimHPCoilData> HeatReclaimSimple_WAHPCoil;
+    extern Array1D<HeatReclaimDataBase> HeatReclaimDXCoil;
```

Comment:
Don't need so many data structs. Base struct is now good for all these types except for condenser.